tool/gravity/cli/rpcagent.go

```diff
@@ -252,37 +254,6 @@ func deployAgents(ctx context.Context, env *localenv.LocalEnvironment, req deplo
 	return clientCreds, nil
 }
 
-func deployUpdateAgents(ctx context.Context, localEnv, updateEnv *localenv.LocalEnvironment, req deployAgentsRequest) error {
```

Comment:
There's no more dedicated `agent deploy` for update - it is using the same code as other cluster operation implementations.